scripts/regenerate-packages

```diff
@@ -6,10 +6,11 @@ cd $(dirname $0)
 if [[ -z ${BRANCH} ]]; then
     branch=$(git rev-parse --abbrev-ref HEAD)
 else
-    echo "Using branch ${BRANCH}"
     branch=${BRANCH}
 fi
 
+echo "Using branch ${branch}"
```

Comment:
Does this variable name need to be capitalized?

Reply:
we copy BRANCH into branch, so the variable we want to use is the lowercase version.

Also, this script is pretty much never going to be used by this repository again; it's just something that gets updated by the automated `make docs` command to keep it in sync with the Go templates in rancher/charts-build-scripts, but it won't affect the functionality of this repository at all.